dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -1,5 +1,8 @@
 FROM docker-config-engine
 
+RUN echo 'deb http://debian-archive.trafficmanager.net/debian jessie-backports main' >> /etc/apt/sources.list && \
```

Comment:
why do we need add backports into the config-engine? which package requires this backport repo?

Reply:
Removed unused.